lib/msf/core/db_manager/db_export.rb

```diff
@@ -0,0 +1,20 @@
+require 'msf/core/db_export'
+
+module Msf::DBManager::DbExport
+  def run_db_export(opts)
+    exporter = Msf::DBManager::Export.new(framework.db.workspace)
+
+    output_file = exporter.send("to_#{opts[:format]}_file".intern, opts[:path]) do |mtype, mstatus, mname|
+      if mtype == :status
+        if mstatus == Msf::DBManager::Export::STATUS_START
+          puts("    >> Starting export of #{mname}")
```

Comment:
The code this was copied from used `print_status` since it was user facing. Should we be using `puts` here or `ilog`?

Reply:
Ahh yeah, `print_status` wasn't defined when I tried to use it here. I meant to come back and change it but forgot.

Reply:
Fixed.